lib/trino-hive-formats/src/main/java/io/trino/hive/formats/line/json/JsonDeserializer.java

```diff
@@ -352,10 +354,8 @@ void decodeValue(JsonParser parser, BlockBuilder builder)
             try {
                 bigDecimal = HiveFormatUtils.parseDecimal(value, decimalType);
             }
-            catch (NumberFormatException _) {
-                // parsing errors are simply ignored and the field is null
-                builder.appendNull();
-                return;
+            catch (NumberFormatException e) {
+                throw new TrinoException(BAD_DATA, "Error Parsing a column in the table: " + e.getMessage(), e);
```

Comment:
Let's change all these to

> Error parsing value

Reply:
That code is outdated, it's since been changed to `"Failed to parse value"`